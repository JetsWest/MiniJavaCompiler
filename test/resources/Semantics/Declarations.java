class SomeClass {
    public static void main(String[] args) {
        System.out.println(new A());
    }
}

class A {
    public int foo() {
        int i;

        x = 1;
        i = y + z;
        i = b.bar();

        return 0;
    }
}