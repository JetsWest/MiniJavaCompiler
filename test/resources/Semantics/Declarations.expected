(Line 11) Variable x not defined in current scope.
(Line 12) Variable y not defined in current scope.
(Line 12) Variable z not defined in current scope.
(Line 12) Variable b not defined in current scope.